components/project-card/project-card-detailed.jsx

```diff
@@ -156,86 +196,124 @@ class DetailedProjectCard extends React.Component {
           <a
             href={twitterUrl}
             onClick={evt => this.handleTwitterShareClick(evt)}
-            className="btn twitter-share d-inline-block align-self-center mx-3"
+            className="circle-twitter large"
           />
         </div>
-      </div>
+      </aside>
     );
   }
 
-  renderTopHeader() {
+  renderTitleAuthor() {
     return (
-      <div className="col-12 mb-3">
-        <div className="row">
-          <div className="col-12 col-sm-8">
-            <Title title={this.props.title} className="mb-1" />
+      <div className="title-author-wrapper mb-4 mb-md-5 mb-lg-0">
+        <div className="title-author">
+          <header className="mb-md-4">
+            <Title title={this.props.title} />
             <Creators
               creators={this.props.relatedCreators}
               showLabelText={true}
               creatorClickHandler={(event, name) =>
                 this.handleCreatorClick(event, name)
               }
+              className="mb-4"
             />
+          </header>
+          <div className="action-panel d-md-flex justify-content-lg-between">
+            {this.renderVisitButton()}
+            {this.renderGetInvolvedButton()}
           </div>
         </div>
       </div>
     );
   }
 
-  renderLeftColumn() {
-    let wrapperClassnames = classNames(`col-12 col-md-8`);
-
+  renderThumbnail() {
     return (
-      <div className={wrapperClassnames}>
-        <Thumbnail thumbnail={this.props.thumbnail} />
-        {this.renderVisitButton()}
-        <Description description={this.props.description} className="mt-3" />
-        <WhyInteresting interest={this.props.interest} />
-        {this.renderTimePosted()}
-        <IssuesAndTags issues={this.props.issues} tags={this.props.tags} />
+      <div className="thumbnail-wrapper mb-4 mb-md-0">
+        <div className="thumbnail-container">
+          <Thumbnail thumbnail={this.props.thumbnail} />
+        </div>
       </div>
     );
   }
 
-  renderRightColumn() {
-    let wrapperClassnames = classNames(`col-12 col-md-4 mt-3 mt-md-0`);
+  renderMainContent() {
+    let getInvolved = classNames("get-involved w-100 mb-5", {
+      "d-none":
+        !this.props.getInvolved &&
+        !this.props.getInvolvedUrl &&
+        !this.props.helpTypes.length > 0
+    });
 
     return (
-      <div className={wrapperClassnames}>
-        {this.renderActionPanel()}
-        <GetInvolved
-          getInvolved={this.props.getInvolved}
-          getInvolvedUrl={this.props.getInvolvedUrl}
-          helpTypes={this.props.helpTypes}
-          sendGaEvent={config => this.sendGaEvent(config)}
-        />
-      </div>
-    );
+			<article
```

Comment:
Use spaces for indentation instead of tabs.

Reply:
I do, I have no clue why it looks like this.